flow/src/main/java/com/sequenceiq/flow/cleanup/FlowCleanupConfig.java

```diff
@@ -9,7 +9,14 @@ public class FlowCleanupConfig {
     @Value("${flowcleanup.intervalhours:24}")
     private int intervalInHours;
 
+    @Value("${flowcleanup.endtime.duration.hours:24}")
+    private int endTimeDurationInHours;
```

Comment:
We can rename it to retentionPeriodInHours.

Reply:
Updated this